src/commands/force/user/password/generate.ts

```diff
@@ -16,6 +15,31 @@ interface PasswordData {
   password: string;
 }
 
+interface PasswordConditions {
```

Comment:
if you export interfaces from core/User, you can import them here, so the interface only exists in one place.